andela/team-odd-bn-frontend
# #69258595 A user should be able to edit travel requests

#### What does this PR do?
This PR allows users to edit travel requests
#### Description of Task to be completed?
- Design the UI as in the mock up
- Setup redux 
#### How should this be manually tested?
Follow the following steps to test this feature
- `git clone https://github.com/andela/team-odd-bn-frontend`
- `git checkout ft-edit-travel-requests-169258595`
- `npm run dev`
- On the left sidebar, click requests
- You will see a list of requests, click view
- Click edit button 
- A pop up will appear
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#### Screenshots (if appropriate)
<img width="1436" alt="Screen Shot 2020-01-30 at 10 53 38" src="https://user-images.githubusercontent.com/45129725/73435598-a356ad00-4351-11ea-82a8-6f1c16b49021.png">

#### Questions:
